src/components/control-component.js

```diff
@@ -18,7 +18,7 @@ import getFieldFromState from '../utils/get-field-from-state';
 import getModel from '../utils/get-model';
```

Comment:
Sorry this diff is hard to read.  I really did not change a lot in this file but due to wrapping almost the entire contents in the 

```
function createControlClass(customControlPropsMap = {}, defaultProps = {}) {

}
```

block with indenting everything inside it makes it look like I changed everything.  Besides that I will point out what else I actually changed with comments.